article/app/views/fragments/emailArticleBody.scala.html

```diff
@@ -1,20 +1,68 @@
-
 @(page: PageWithStoryPackage)(implicit request: RequestHeader)
 
-@import common.LinkTo
-@import views.BodyCleaner
-@import views.support.TrailCssClasses.toneClass
-@import conf.switches.Switches._
+@import views.support.EmailArticleImage
+@import views.support.Zurb._
+@import model.liveblog._
+@import model.EmailAddons.EmailArticle
 
 @defining(page.article) { article =>
-    @fragments.headTonal(article, article.commercial.isAdvertisementFeature && NewCommercialContent.isSwitchedOn, showMeta = false)
+    @fullRow {
+        <img src="@article.banner">
+    }
+
+    @paddedRow {
+        <h1>@article.trail.headline</h1>
+    }
 
     @fragments.emailMainMedia(article)
 
-    @fragments.contentMeta(article, showExtras = false)
+    @paddedRow(Seq("author")) {
+        @article.trail.byline.map { byline =>
+            <h3 class="byline">@byline</h3>
+
+        @if(article.content.tags.contributors.length == 1) {
```

Comment:
Could you use a `map` here or does it just have to be if there is just one contributor?